packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -370,6 +370,31 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
       }
 
       this.context.updateClientState(clientState);
+
+      /* @conditional-compile-remove(call-transfer) */
+      const acceptedTransferCallState = this.context.getState().acceptedTransferCallState;
+
+      /* @conditional-compile-remove(call-transfer) */
+      // If there has been an accepted transfer call that is now in the connected state we should ensure we leave the current call.
```

Comment:
What does the SDK do to the current call? Is it placed on hold or do they leave the user also connected? (this feels like something that should be handled at the SDK layer)

Reply:
They leave the user connected. So there is actually 2 calls with a connected state.

Reply:
Can you confirm with calling if this is expected? I struggle to see the scenario where you want to be active in two separate calls at once. If it is expected (to be in `Connected` with two calls at once), can calling SDK provide a snippet on how developers are expected to handle this? e.g. Should the developer manually go `on hold` in one call?

Reply:
So Teams user perform a safe transfer which is "wait for transfer completion to end if success and continue to original call if failure". So on the latter when the transfer request times out, the original call is still connected. The ACS user accepts the transfer and essentially makes an outgoing call to the transfer target.
There is no mention of holding as transferee from the documentation
![image](https://github.com/Azure/communication-ui-library/assets/79475487/2cf40562-6d81-4993-a6e5-f164458b9225)
From https://learn.microsoft.com/en-us/azure/communication-services/how-tos/calling-sdk/transfer-calls.
I think even if I hold the original call, I would still have a similar problem because we have 1 call in hold state and the other in connected state.